doc/faq/howto_faq.rst

```diff
@@ -423,18 +423,12 @@ locators as desired because the two axes are independent.
 Generate images without having a window appear
 ----------------------------------------------
 
-The easiest way to do this is use a non-interactive backend (see
-:ref:`what-is-a-backend`) such as Agg (for PNGs), PDF, SVG or PS.  In
-your figure-generating script, just call the
-:func:`matplotlib.use` directive before importing pylab or
-pyplot::
-
-    import matplotlib
-    matplotlib.use('Agg')
-    import matplotlib.pyplot as plt
-    plt.plot([1,2,3])
-    plt.savefig('myfig')
+Simply do not call `~matplotlib.pyplot.show`, and directly save the figure to
```

Comment:
This won't work in an interactive terminal if we are in 'interactive' mode (as we call `show()` implicitly on figure creation).

Reply:
Fair enough (that's not a use case I thought about).  Do you want to recommend calling ioff(), or use('agg')?  (I admit I never ever use ioff(), so heh.)